tests/kernel/virtio_console.pm

```diff
@@ -7,7 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 #
-# Summary: Log into and run a script on a virtio serial terminal
+# Summary: Stress test the virtio serial terminal
```

Comment:
Can you add a bit from the commit message here to describe the purpose of the test?